configure.py

```diff
@@ -777,6 +777,10 @@ def check_compiler(o):
     # to a version that is not completely ancient.
     warn('C compiler too old, need gcc 4.2 or clang 3.2 (CC=%s)' % CC)
 
+  # Fix based on work of @bnoordhuis from 
+  # https://github.com/nodejs/node-v0.x-archive/issues/4186 .
+  o['variables']['is_clang'] = 1 if is_clang else 0
```

Comment:
```suggestion
  o['variables']['clang'] = 1 if is_clang else 0
```
To fit with usage in other places.